docs/guides/metrics/starter-kit/README.md

```diff
@@ -0,0 +1,107 @@
+<!-- Header block for Apache DevLake Starter Kit -->
```

Comment:
@ddalton-jpl - can we use the standard template instead of this format? Unless you plan to propose this `README.md` be deployed as a Git repository, its best if we follow the standard SLIM best practice guide template format:
1. Leverage our standard template: https://nasa-ammos.github.io/slim/docs/contribute/submit-best-practice#use-our-standard-guide-template
2. Move the `README.md` file to `/metrics` as that guide should be the first thing readers see, and should link to all relevant markdown files / artifacts.

Reply:
Hey @ddalton-jpl - awesome progress! I would suggest we include your guides into a new sub-folder within the ["Software Lifecycle" directory](https://nasa-ammos.github.io/slim/docs/category/software-lifecycle) called "Metrics". By the way, if you include a `_category_.json` file within your folder (see [this example](https://github.com/NASA-AMMOS/slim/blob/main/docs/guides/software-lifecycle/application-starter-kits/_category_.json)), you can ensure that the folder name that appears on the website is exactly what you'd like to call it. I'd suggest something general right now like "Metrics" that's technology agnostic.

Reply:
@riverma Okay, I moved it